src/services/sessions.js

```diff
@@ -1131,16 +1131,7 @@ module.exports = class SessionsHelper {
 					responseCode: 'CLIENT_ERROR',
 				})
 			}
-			// check if the session is accessible to the user
```

Comment:
@adithyadinesh0412 can you please check why this is removed. You can compare with latest develop code